### ISSUE TITLE
execCommand browser incompatibilities

### ISSUE THREAD

**Issue by njdjacobs** [w3c/editing#289]
The spec aims to follow current browser implementations, with notes explaining incompatibilities, but some of this material is outdated. For example, it is no longer usual to flag issues with Internet Explorer, except perhaps to mention that IE's implementation is incompatible. A note to section 9.21 about different line-breaking behaviours in different browsers has been superseded: all current browsers converged on <div> more than five years ago. The note could be deleted.

In my opinion, the statement in the intro that browsers copied the APIs "in a haphazard and imprecise fashion" is unfair to current browser implementors because they appear to have largely converged since that was written. I suggest it be removed.

**Comment by johanneswilm** [comment]
> The spec aims to follow current browser implementations, with notes explaining incompatibilities, but some of this material is outdated. 

I agree. Would you want to create a PR updating/removing all the notes that are concerned with current browser implementation status?

**Comment by johanneswilm** [comment]
> In my opinion, the statement in the intro that browsers copied the APIs "in a haphazard and imprecise fashion" is unfair to current browser implementors because they appear to have largely converged since that was written. I suggest it be removed.

This is prose written by the original author AFAIK. It was written at a time when it seemed like `execCommand` would be the future of text editing in the browser and this document was supposed to help them with that. There was a felt need to state it, because implementations differed.

I would be OK with removing the introduction entirely or changing the title to something like "historical background and context" or similar.
